### ISSUE TITLE
Can't find next file in playlist

### ISSUE THREAD

**Issue by ForSquirel** [jellyfin/jellyfin#2844]
**Logs**
[2020-04-12 12:57:37.242 -04:00] [WRN] HTTP Response 204 to "192.168.1.15". Time (slow): 0:00:50.9064502. "http://192.168.1.7:8096/Sessions/Playing/Stopped"

I was able to recreate with different albums

**Server System (please complete the following information):**
 - OS:Rasbian Buster
 - Jellyfin Version: 10.5.3
 - Hardware settings & device: Orange Pi
 - Reverse proxy: nginx (same occurs on local interface(
 - Other hardware notes: 

**Client System (please complete the following information):**
 - Device: Android App, Brave web Browser
 - OS: Android 7, Ubuntu 18.04
 - Client: Web and Android
 - Browser (if Web client): Brave
 - Client and Browser Version: [e.g. 10.3.4 and 68.0]

I only recently updated. Older version was rock solid. I rebuilt my database and am still getting same error.

**Comment by Artiume** [comment]
Could you check your browser console and see if there's any errors or clues. Could you also enable debug logging for the jellyfin server. 50 seconds is a long wait, I wonder if it could be some sort of database performance issues?

**Comment by ForSquirel** [comment]
browser isn't showing anything, debug is the same. 

Not sure what's happened. As soon as I start playback the CPU gets pegged to the point of the system being almost unresponsive and it takes 12-15 seconds to load the next song.

When I first installed everything i could login and be up and playing rather quickly. The only thing I've changed recently was throwing up a quick mindlna server to share pictures.

Also, I am not using Jellyfin for anything other than audio. Other than myself I have 1 other user. The issues I'm having are all on LAN. Like I said in the OP. Everything has been rock solid since install. I'm not sure what version I was using because I just haven't updated the system till last week and this morning noticed the issues.
